lstchain/scripts/longterm_dl1_check.py

```diff
@@ -745,8 +750,21 @@ def plot(filename='longterm_dl1_check.h5'):
     fig_ped_stddev.y_range = \
         Range1d(0.,1.1*np.max(runsummary['ped_charge_stddev']))
 
+    frac = runsummary['num_pedestals_after_cleaning'] / \
+           runsummary['num_pedestals']
+    err = np.sqrt(frac*(1-frac)/runsummary['num_pedestals'])
+    fig_ped_clean_fraction = \
```

Comment:
minor style thing, but line continuations are generally discouraged. Use the implicit line continuation inside parentheses, e.g.:

```
fig_ped_clean_fraction = show_graph(
    x=...,
    y =...
)
```

Reply:
But in some cases like this one readability is not so good... line is broken at the square bracket (!)

Reply:
At which square brackets? This is a Function call. This is how I (and I think black) would format it:

```
    fig_ped_clean_fraction = show_graph(
        x=pd.to_datetime(runsummary["time"], origin="unix", unit="s"),
        y=frac,
        xlabel="date",
        ylabel="Fraction of pedestals surviving cleaning",
        ey=err,
        xtype="datetime",
        ytype="linear",
        point_labels=run_titles,
    )

```

Reply:
I meant the automatic line breaking by pycharm (probably configurable?) - but your suggestion is fine, will do.